H2 pcb

g0DTuRtI3 opened the pull request.

efondere committed:
h2-pcb: create project

Created project from template v1.1.0

g0DTuRtI3 committed:
Added Manometer, Actuator, Temperature Sensor

g0DTuRtI3 committed:
Added Mosfets and LEDs

efondere commented on h2-pcb/h2-pcb.kicad_sch:
```diff
@@ -0,0 +1,13572 @@
+			)
+			(justify right bottom)
+		)
+		(uuid "66f78937-df8a-4ae4-9d12-d6de7e7eb59f")
```
Les _labels_ `NC_1`, `NO_1`, etc. ne sont pas connectées à des pins GPIO du stm32. Il faut faire comme les autres capteurs avec des _hierarchical labels_ (en input).

efondere commented on h2-pcb/h2-pcb.kicad_sch:
```diff
@@ -0,0 +1,13572 @@
+			(justify right bottom)
+		)
+		(uuid "acea20cf-0204-4e3e-a7e9-137b7940f1ca")
+	)
```
**Signaux des MOSFETs**:
Pour chaque circuit de MOSFET, il devrait y avoir deux signaux (`n` vaut 1, 2 ou 3):
- `ACTUATOR_GPIOn`: connectant les GPIO du stm32 à l'endroit où se situe actuellement les signaux `MOSFET_n` des circuits de MOSFET
- `ACTUATOR_CTRLn`: reliant la sortie des MOSFET (là où se trouvent les résistances R17, R22 et R27) au connecteur des actuateurs (là où il y a le label `MOSFET_n`)

efondere replied:
2025-07-25: il faut connecter les signaux `ACTUATOR_CTRLn` directement au drain (actuellement source, voir mon autre commentaire) des PMOS, pas besoin des résistances R17, R22 et R27 ni des GND.

efondere commented on h2-pcb/h2-pcb.kicad_sch:
```diff
@@ -0,0 +1,13572 @@
+			(project "h2-pcb"
+				(path "/4b4cf3aa-c94c-4150-b9c5-35049cd71136"
+					(reference "R24")
+					(unit 1)
+				)
+			)
+		)
+	)
+	(symbol
+		(lib_id "Simulation_SPICE:PMOS")
+		(at 55.88 133.35 0)
+		(unit 1)
+		(exclude_from_sim no)
+		(in_bom yes)
+		(on_board yes)
+		(dnp no)
+		(fields_autoplaced yes)
+		(uuid "bbda000e-b0c4-4166-8cb7-e6922c3a84a5")
+		(property "Reference" "Q2"
+			(at 62.23 132.0799 0)
+			(effects
+				(font
+					(size 1.27 1.27)
+				)
+				(justify left)
+			)
+		)
+		(property "Value" "PMOS"
+			(at 62.23 134.6199 0)
+			(effects
+				(font
+					(size 1.27 1.27)
+				)
+				(justify left)
+			)
+		)
+		(property "Footprint" ""
```
La source et le drain des PMOS sont inversés. **Rappel**: la source est toujours du côté _stable_ (où on connait toujours la tension). Dans notre cas, la source (des PMOS seulement) devrait être côté +24V (vers le haut) et le drain vers le bas, connecté au signal qui se rend jusqu'à un actuateur (à travers le connecteur; voir mon autre commentaire sur les signaux des MOSFETs).

efondere replied:
2025-07-25: Je pense que tu as oublié d'inverser la source et le drain des PMOS

efondere commented on h2-pcb/h2-pcb.kicad_sch:
```diff
@@ -0,0 +1,13572 @@
+					(size 1.27 1.27)
+				)
+				(hide yes)
+			)
+		)
+		(property "Description" "P-MOSFET transistor, drain/source/gate"
+			(at 55.88 133.35 0)
+			(effects
+				(font
+					(size 1.27 1.27)
+				)
+				(hide yes)
+			)
+		)
```
As-tu trouvé un modèle pour les MOSFET? Ça nous permettrait de savoir ce qu'on a besoin pour les résistances R16/R15, R20/R21, R25/R26. Tu peux le chercher sur JLCPCB en essayant de matcher les charactéristiques qu'il nous faut (il faut au moins +24V comme tension maximale entre source et drain, mais il n'y aura pas beacoup de courant qui va passer)

efondere replied:
2025-07-25: Tel que discuté pendant la réunion, il faut trouver un autre MOSFET que ceux des démos. Quand tu auras trouvé, tu mettras le lien dans le wiki et on pourra réajuster les valeurs des résistances qui servent de diviseurs de tension au besoin.

efondere commented on h2-pcb/h2-pcb.kicad_sch:
```diff
@@ -0,0 +1,13572 @@
+		)
+		(uuid "08ffabb0-e1f3-4a78-bac2-6679b0339183")
+	)
+	(hierarchical_label "PB2"
```
Quand je roule le ERC, je remarque qu'il y a plusieurs erreurs au niveau des _hierarchical labels_: je pense que tes pins ont été désynchronisées. Supprimes celles que tu vois dans la page "Root" et rajoute-les avec le bouton "Place sheet pins" à nouveau. Tu peux rouler le ERC avant et après pour vérifier que ça marche. Aussi, profites-en pour changer la direction des pins: tout devrait être en input sauf pour les pins des LEDs.

efondere replied:
2025-07-25: Quelques trucs pour résoudre les ERC:
- Le warning pour `ACTUATOR_CTRLn` prenant le nom de GND: ça va disparaître quand tu feras les changements que j'ai décris dans un autre commentaire!
- Pour `MOSFET_n`: tu peux simplement les supprimer (avec les hierarchical pins PA7-PA9)
- Pour "pin not connected": il faut remettre les "x" sur les pins PC4-PC6 du microcontrôleur
- Pour les autres hierarchical labels: je te conseille de les supprimer dans "STM32 Base Layout" à la première page et de les ré-ajouter avec l'outil "place sheet pins"; ça va mettre à jour les directions en même temps!

g0DTuRtI3 committed:
h2-pcb: implement review comments in schematic

efondere commented:
2025-07-25: Pourrais-tu renommer ton dernier commit (`git commit --amend` ou en utilisant ton éditeur de code; assure toi de pas avoir de fichiers _staged_ (i.e. après `git add`) quand tu fais ça) pour quelque chose comme: `h2-pcb: implement review comments in schematic` stp? Idéalement tu pourrais même faire une liste des changements dans la description du commit! C'est normal d'avoir à force push (sur ta branche) après ça. C'est une bonne pratique git, mais si tu as trop de misère, n'hésite pas à m'écrire! Aussi, je vais remettre la PR en draft parce qu'on va seulement merge à la toute fin

efondere commented:
2025-07-25: Voir mes réponses à mes précédents commentaires (commençant par "2025-07-25"), une fois que c'est fait, n'hésite pas à me faire réviser à nouveau si tu as des hésitations! La prochaine étape c'est de rajouter les footprints et une entrée JLCPCB avec le numéro de la pièce dans les propriétés de chaque symbole. Tu peux aller voir ce qu'on a fait dans le template comme exemple!

g0DTuRtI3 committed:
h2-pcb: implement review comments in schematic 2.0

g0DTuRtI3 committed:
Added footprints and jlc part numbers

efondere commented on h2-pcb/Actuator.pretty/Actuator_C200714.kicad_mod:
Tu peux ajouter le symbole à la librarie de _footprints_ `Exocet` qui existe déjà. Aussi, tu pourrais le nommer quelquechose comme `TE_UNIVERSAL-MNL_6Pin_Horizontal_PCB_Header` étant donné qu'il n'est pas vraiment spécifique aux actuateurs.

efondere commented on h2-pcb/Actuator.pretty/Actuator_C200714.kicad_mod:
```diff
@@ -0,0 +1,184 @@
+(footprint "Actuator_C200714"
```
Réfère-toi à la section PCB Footprint/symbol de la [page JLCPCB](https://jlcpcb.com/partdetail/TEConnectivity-3504311/C200714) pour voir où placer les lignes sur la layer de silkscreen. Pas obligé de toutes les mettre, mais au moins qu'on puisse avoir une idée de la taille du connecteur autour des pads. Aussi, plutôt que d'utiliser des cercles pour indiquer l'épaisseur de la trace, tu devrais changer le diamètre du pad lui-même (pour avoir un cercle rouge autour du drill hole). Un diamètre de 3.5mm pour le pad et 1.78 pour le drill hole devraient faire l'affaire. Pour la pin 1, je conseille aussi de choisir un pad shape `Rounded rectangle` à la place pour qu'on puisse facilement l'identifier.

g0DTuRtI3 committed:
h2-pcb: implement review comments in footprint

efondere commented:
TODO: ajouter fusible au 24V. Faire la même chose que pour le 5V dans "Network connector" à la page 2. Assumer une valeur de 5A pour le fusible.

g0DTuRtI3 committed:
h2-pcb: Added fuse to +24V network connector pin in schematic

Added a safety fuse of 5A to the +24V network connector pin

g0DTuRtI3 committed:
h2-pcb:Added footprint to 5A Fuse

sylvainrarnaud-magic committed:
Début du routage PCB Hydrogène : placement et début de routage.

efondere commented on h2-pcb/h2-pcb.kicad_sch:
TODO (pour moi): valider avec l'équipe d'énergie qu'on a la bonne quantité d'actuateurs et manomètres.

efondere commented on h2-pcb/h2-pcb.kicad_sch:
J'ai pensé à ça, et je pense qu'on devrait ajouter des condensateurs de découplage près des actuateurs pour supporter les "peak" de demande sans trop impacter le reste des composants.

Il est difficile de dire combien de temps ces "peak" peuvent durer, mais idéalement un condensateur permettra de réduire la demande instantannée au reste du circuit. Quelque chose comme $1000 \micro F$ pourrait faire l'affaire. Ça sera un condensateur électrolytique (polarisé), donc il faudra faire attention à la polarité.

C'est une petite modification au schéma, mais n'hésitez pas à demander de l'aide dans le chat discord à @g0DTuRtI3 ou moi!

efondere replied:
Désolé, finalement je pense que 100 uF, ça sera encore trop. On peut les changer pour des 10 uF, sinon on risque d'avoir un gros "peak" de courant quand on alimente le PCB initialement. Aussi, il faut les mettre entre +24V (pin 6) et le GND (pin 5).

efondere commented on h2-pcb/h2-pcb.kicad_pcb:
Pour commencer, je vois que vous avez essayé de coller le plus possible les composants. Ce n'était pas nécessaire et malheureusement, vous avez bougé quelques traces qui devaient rester symmétriques pour des raisons d'interférences électromagnétiques (je ferai des commentaires séparées pour chaque). À la fin, ça a du être très compliqué pour vous de tout connecter!

Pour vous aider, vous pouvez commencer en élargissant de manière exagérée la taille du PCB. Une fois tous les composants placés, vous pouvez ensuite la réduire à la taille nécessaire pour que les composants entrent correctement, sans être trop regroupés. De manière générale, il est préférable que le PCB soit bien organisé que petit. Quand on va recevoir le PCB ça va aider pour  régler des bugs ou comprendre si quelque chose ne marche pas (sinon, il faut tenter de retrouver la pièce parmis 20 autres!) C'est très important de pouvoir associer le numéro des pièces (R12, C14, Q5, etc.) sur le PCB au bon composant.

La première chose que vous pouvez faire, c'est de réorganiser le placement des pièces pour que ça soit plus facile de suivre chaque partie du schéma (i.e. les MOSFETs près des actuateurs, résistances _pull-up_ près des capteurs de rempérature etc.). Une fois que c'est fait, faites un commit et je pourrai valider le placement et que vous vous évitiez de devoir re-router une grande partie du PCB.

efondere replied:
Voici par exemple un ancien PCB où on avait utilisé le Silkscreen pour nous aider à visualiser les différentes parties du PCB:
<img width="1261" height="1163" alt="image" src="https://github.com/user-attachments/assets/a6519258-accc-4e0d-a1d5-53e069534bf7" />

Comme vous le voyez, ça permet aussi de simplifier le passage des traces entre les composants!

efondere commented on h2-pcb/h2-pcb.kicad_pcb:
Je vois à quelques endroits que vos traces ont des angles très abruptes. De manière générale, on essaye de garder les traces les plus droites possible, et seulement changer de direction lorsque absolument nécessaire. Les électrons sont un peu comme un cours d'eau à cette échelle et des virage serrés peuvent les ralentir (i.e. ajouter de la résistance au circuit). Si un angle est vraiment nécessaire, il faudrait qu'il soit au maximum à 45 degrés.

C'est pas quelque chose dont j'ai parlé pendant la formation étant donné que c'est un concept plus avancé, mais c'est une bonne chose à savoir tout de même!

efondere commented on h2-pcb/h2-pcb.kicad_pcb:
# Signaux différentiels

Comme mentionné dans mon premier commentaire, il y a quelques traces qui doivent être le plus parallèle possible et/ou de même longueur. Vous allez comprendre l'importance de ces facteurs dans des cours de 2e session de génie élec', mais grosso modo ça permet de s'assurer que le bruit externe affecte les signaux de la même manière.

De manière générale, l'information est transmise par des différences de potentiel entre deux fils. Si leur potentiel est augmenté au diminué de manière équivalente, alors la différence reste la même et le signal passe correctement. Par contre, si un fil est affecté différemment par le bruit ou possède une résistance plus grande, alors il y a un déséquilibre et le signal risque d'être mal transmis.

Voici les endroits où sa s'applique:

## CAN_H / CAN_L
<img width="766" height="268" alt="image" src="https://github.com/user-attachments/assets/b4ae00e1-3ea1-4dd4-bfb6-9a6889ae1ad3" />
Les deux traces devraient se rendre en ligne droite.

## Osillateur en crystal
<img width="504" height="744" alt="image" src="https://github.com/user-attachments/assets/fbcc5319-5be0-47d4-b73e-bc743ddbc9ce" />
Les deux traces devraient être symmétriques et de même longueur.

## VCP_TX / VCP_RX
<img width="991" height="264" alt="image" src="https://github.com/user-attachments/assets/fbe52298-0896-4524-aa30-4c27bbbd7393" />
J'ai surligné les traces en question, désolé ça se voit un peu mal (elles ont l'air vertes). Il faudrait que ces traces soient le plus possible de la même longueur (vous pouvez voir cette info dans la barre tout en bas de Kicad en cliquant sur un des segment de chaque trace normalement; pour moi c'est écrit "Routed Length" -- et pas "Segment Length"!). Le serpentin permet justement d'allonger une des deux traces pour l'équilibrer.

Allez voir ce qui a été fait pour le template (vous devriez avoir un dossier tempalte avec un fichier `.kicad_pcb`) pour voir comment placer les pièces, les vias, etc.


Encore une fois, c'est un concept plus avancé dont je n'ai pas parlé dans la vidéo, je m'en excuse!

efondere commented on h2-pcb/h2-pcb.kicad_pcb:
Je vois à quelques endroits que des condensateurs de découplages ont été déplacés. C'est l'un des rares composants qui doit obligatoirement être très près du microcontrôleur. Ça permet de réduire l'impact d'une demande soudaine de courant par le microcontrôleur sur le reste du système. Voir: https://en.wikipedia.org/wiki/Decoupling_capacitor .

Fiez-vous encore une fois au template pour savoir quels condensateur ça impacte (il n'y en a pas beaucoup je crois).

Encore une fois un concept plus avancé qui n'avait pas été mentionné dans les formation, désolé pour ça!

efondere commented on h2-pcb/h2-pcb.kicad_pcb:
En roulant le DRC, je vous qu'il y a quelques endroits du PCB qui ne sont pas connectés. En fait, il y a des endroits où le ground plane est isolé du reste, par exemple ici:
<img width="667" height="658" alt="image" src="https://github.com/user-attachments/assets/7c50d4c5-479d-44ec-b7b4-56627ab4c92d" />

KiCad vous montre aussi ce qui n'est pas connecté à l'aide de petites lignes bleues pour que vous puissiez voir plus facilement ce qu'il reste à brancher. Vous pouvez passer à travers la liste des erreurs DRC à la fin pour vous assurer que vous n'avez pas oublié de connexions.

Malheureusement, il n'y a pas de solution pour ça autre que de refaire certaines autres traces pour ne pas isoler certaines parties du PCB. Par exemple ici, vous pouvez faire passer le signal NC_2 sur la couche de dessous avec un via, puis contourner les traces du dessous et laisser le bout de ground plane se connecter au reste. Pour aller plus vite, vous pouvez appuyer sur `V` pour changer de layer sur le PCB (ça fait automatiquement un via quand vous routez une trace).

efondere commented on h2-pcb/h2-pcb.kicad_pcb:
Pas mal tous les warnings du DRC sont liés au silkscreen. Pour rappel, le silkscreen c'est le texte qui sera imprimé sur le PCB: on y retrouve le logo d'Exocet, la référence des composants (R12, C14, Q3, etc.), un contour pour les pièces à placer, le nom des différents composants qui doivent être branchés dans les connecteurs, etc. Comme vous avez essayé de tout regrouper près du microcontrôleur, vous avez du avoir de la misère à placer le texte des référence aux bon endroits et il y a donc beaucoup de overlap. En particulier KiCad se plaint que:
- Le texte des référence est un par-dessus l'autre
- Le texte est par-dessus un "soldermask" c'est à dire là où seront soudés les composants -- donc on ne le verra pas!

En réorganisant le placement des pièces, vous allez voir que ces warning vont disparaître assez rapidement!h Vous pourez corriger ce qui reste à la fin.

efondere reviewed (commented):
2025-11-05: Allez-y une chose un commentaire à la fois et n'hésitez surtout pas à faire des commit à chaque fois et confirmer avec moi à chaque étape, ça vous évitera de devoir revenir en arrière et c'est plus facile pour moi de réviser par petits bouts. Si je n'ai rien manqué, une fois tous mes commentaires résolus, le PCB sera à 99% prêt d'être commandé (il risque de rester quelques erreurs du DRC un peu random qu'on pourra résoudre rapidement, c'est normal).

victoriagobjila9-jpg committed:
Mise à jour du PCB : déplacement des composantes

victoriagobjila9-jpg committed:
misc: add .DS_Store to gitignore

victoriagobjila9-jpg committed:
h2-pcb: add manometer, actuator and ds2428

efondere committed:
h2-pcb: revert pcb layout to template

victoriagobjila9-jpg committed:
Mosfets and Actuators dans truc pcb

victoriagobjila9-jpg committed:
Ajout des footprints aux composantes & update du pcb (routage almost done ?)

victoriagobjila9-jpg committed:
Update sur le routage, silkscreen, jlcpcb parts #

g0DTuRtI3 committed:
h2-pcb: Fixed DRC errors + cleaned PCB

victoriagobjila9-jpg committed:
Footprint to DS24 assigned

g0DTuRtI3 committed:
h2-pcb : Added DS2484 footprint

efondere commented on h2-pcb/h2-pcb.kicad_sch:
Il faudrait ajouter des test points (copier ceux de la page STM32) sur les nets suivants:
- SDA_TEMP
- SCL_TEMP
- ONEWIRE
- MANOMETER_X (X=1,2,3)

Aussi, il faut les garder en "Do not populate" comme sur la page STM32.

efondere committed:
h2-pcb: fix DRC warnings (islands + clearance)

There was an issue with the clearance on the pins of the voltage
regulator from the template that appeared. The global settings were
overriden for that particular footprint only.

Some traces were re-routed to avoid copper islands on the GND net. There
are no more nets to connect now.

A "sub"-plane was added for a +5V-net near the temperature sensor
connectors to simplify the routing around that area. The +3.3V-net was
not needed there anyways.

efondere committed:
h2-pcb: rename PA7 label to PB13

The pin function for PB13 on the MCU was modified, but the hierarchical
label wasn't updated. The one that used to be called "PA7" was renamed
to "PB13" and connected to that pin.

Also updated the routing to refelct these changes.

efondere commented on h2-pcb/h2-pcb.kicad_pcb:
Vous pouvez écrire le noms des pièces (i.e. Manomètre 1/2/3, Actuator 1/2/3/4, Temp. 1/2/3) à côté des footprints pour qu'on puisse savoir où connecter chaque composant. Pas obligé d'écrire tout au long, on peut se servir d'abbréviations ou autres pour sauver de la place, tant que c'est bien documenter pour ceux qui feront la programmation.

efondere commented on h2-pcb/h2-pcb.kicad_pcb:
Vous ferez attention à la trace du +24V: elle fait le tour du PCB! Comme elle risque de porter beaucoup de courant, il faudrait idéalement:
- La grossir (rendre plus épaisse), voir la convertir en plane (option suggérée)
- Faire des "branches" plutôt qu'une longue chaîne

Pour vous aider, on peut changer l'orientation des connecteurs pour que les pins soient alignées de chaque côté.

efondere commented on h2-pcb/h2-pcb.kicad_pcb:
Essayer de garder le plus parallèle et de même longueur possible les traces pour le I2C (SCL et SDA). Vous pouvez utiliser l'outil "Tune length of single track" à la fin pour ajuster la longueur de la plus courte.

<img width="341" height="426" alt="Image" src="https://github.com/user-attachments/assets/848b9054-7d05-4520-9920-13c91698ffcc" />

efondere commented on h2-pcb/h2-pcb.kicad_sch:
On pourra modifier la cartouche pour ajouter vos noms dans les auteurs, la version etc. quand on va produire la version pour la commande!

efondere reviewed (changes requested):
Dernier set de commentaires je crois! Après ça il ne restera qu'a valider les pièces JLCPCB, leur footprints et on sera prêt à commander!

efondere committed:
h2-pcb: update contactor capacitors

- Changed valued to 10uF
- Changed JLCPCB part
- Connect with +24V instead of +3.3V and update pcb routing

efondere committed:
h2-pcb: add test points to schematic

- Updated PCB, but not routed yet

efondere committed:
h2-pcb: add fuse value to PCB silkscreen

efondere committed:
h2-pcb: update metadata and silkscreen

- Update author, pcb code, version and date metadata
- Add version and code to PCB name in silkscreen

efondere committed:
h2-pcb: make i2c traces parallel

efondere commented:
Update pour DS2484: voir #31

victoriagobjila9-jpg committed:
Ajout des noms des composantes, routage des test points, 24V plane pour les manometres,  mounting holes 4mm de distance du edge

efondere committed:
h2-pcb: replace ds2484 symbol

The symbol is now the one from the official KiCad library. PCB was also
updated and any remaining DRC errors were fixed.

efondere commented on h2-pcb/h2-pcb.kicad_sch:
Mettre les fuse holder en "Do Not Populate" (dans _Properties_), ils coûtent trop cher à faire installer par JLCPCB. Enlever le numéro JLCPCB au passage.

efondere commented on h2-pcb/h2-pcb.kicad_pcb:
Plusieurs textes sur le _silkscreen_ sont trop petits. Des warnings apparaissent dans le DRC, et il faudra donc augmenter leur _width_ et _height_ à au moins 1mm et leur _thickness_ à 0.153 au minimum.

efondere commented on h2-pcb/h2-pcb.kicad_sch:
R10, R11 et R12: remplacer par la même résistance que R7:
- Footprint = 0603
- JLCPCB = C23212
- Valeur = 6.8k

efondere commented on h2-pcb/h2-pcb.kicad_sch:
J8, J9, J10: Mettre en "Do not Populate"

efondere reviewed (commented):
Après avoir essayé de passer la commande sur JLCPCB, voici les changements à faire pour le PCB hydrogène.

efondere commented on h2-pcb/h2-pcb.kicad_pcb:
La diode D3 (rouge) n'a pas le bon footprint. Devrait être 0603 pour matcher la pièce JLCPCB.

g0DTuRtI3 committed:
h2-pcb: implement review comments

efondere commented on h2-pcb/h2-pcb.kicad_sch:
Changer le footprint des connecteurs JST-PH pour capteurs de temperature a des connecteurs horizontaux (JST_PH_S3B-PH-k_1x03_P2.00mm_Horizontal). Garder en "Do not Populate"

Il faudra ensuite les deplacer pour etre sur le bord du PCB (ex. tout a gauche)

efondere committed:
h2-pcb: add plane for left actuator connectors

g0DTuRtI3 committed:
h2-pcb: Add 24V zone for connectors, change footprint for temp sensors + connect

Made 24V zone for right actuators connectors, network connector and 24V fuse holder.
Changed the temperature sensor connectors to JST_PH_S3B-PH-k_1x03_P2.00mm_Horizontal and placed them on the left side of pcb.
Followed with connecting the temperature sensors on the pcb.
TODO : fix 5 errors in DRC.

g0DTuRtI3 committed:
h2-pcb: Fix DRC errors

Rotated the right actuator connectors and moved the zones to include the 3.3V zone with the rest of its zone. Added a zone for the Net J1 24V.

gumile committed:
h2-pcb: change value of LED resistors

gumile committed:
h2-pcb: add fuse holder 01110501Z

gumile committed:
h2-pcb: make edge cuts rectangle and rounded

gumile committed:
h2-pcb: flip actuator connectors 3 & 4 to front layer

gumile committed:
h2-pcb: add DNP to actuator connectors

gumile committed:
h2-pcb: add fuse holder 01110501Z for 5A

efondere committed:
h2-pcb: route some traces away from MCU

efondere committed:
h2-pcb: change MOSFET pins and part number

The orientation of the footprint did not match that of the actual
package. Also, the assembled parts have been changed.
